Lib/test/test_concurrent_futures/executor.py

```diff
@@ -55,8 +57,41 @@ def test_map_exception(self):
         i = self.executor.map(divmod, [1, 1, 1, 1], [2, 3, 0, 5])
         self.assertEqual(i.__next__(), (0, 1))
         self.assertEqual(i.__next__(), (0, 1))
-        with self.assertRaises(ZeroDivisionError):
+
```

Comment:
This test looks fragile to me, +1 to remove test altogether, the fix is straightforward enough